packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1326,6 +1327,9 @@ class RenderPhysicalModel extends _RenderCustomClip<RRect> {
   static final Paint _transparentPaint = new Paint()..color = const Color(0x00000000);
 
   @override
+  bool get alwaysNeedsCompositing => Platform.isFuchsia;
```

Comment:
Hmm, perhaps I should have this ask SceneBuilder instead so the logic and reasoning is centralized.